### DISCUSSION TITLE
Hide/disable the review thumbnails at the top of "Live" page?

### DISCUSSION THREAD

**Discussion by thuesen** [blakeblackshear/frigate#13824] [Ask A Question]
Hi,

On the default startpage ("Live") of the Frigate UI, I see all my (two) cameras. Above them, I see small thumbnails with "reviews".  Can I disable this row, without affecting events being saved? 

I don't need to review stuff or mark as review. But I like to be able to go back and seek out events, in case something should happen.

I've read the config reference and I'm thinking maybe if I add some labels (that will never occur) to be reviewed, it might disable it. But I'm not sure if this has an overall affect on other areas of Frigate.

Thanks in advance.

**Comment by hawkeye217** [comment]
It's not possible to disable the "filmstrip" of reviews on the Live dashboard. However, you can have it display thumbnails rather than looping videos by switching off the "Play Alert Videos" in Settings.

You can also quickly dismiss any of them by clicking the "mark as reviewed" icon.

**Reply by thuesen** [reply]
Thanks, I guess that's a beginning :)

Hope they will include an option to hide/disable the filmstrip eventually.
